src/PluginsDataViewCtrl.cpp

```diff
@@ -0,0 +1,649 @@
+#include "PluginsDataViewCtrl.h"
+
+#include <wx/renderer.h>
+
+#include "PluginDataModel.h"
+
+BEGIN_EVENT_TABLE(PluginsDataViewCtrl, wxDataViewCtrl)
+   EVT_CHAR(PluginsDataViewCtrl::OnCharEvent)
+END_EVENT_TABLE()
+
+#if defined(wxHAS_GENERIC_DATAVIEWCTRL) || defined(__WXGTK__)
```

Comment:
> Ideally this #if would be false, so that e.g.

Ideally, yes. But `wxWidgets` controls are not always have same behavior across platforms. There are three different underlying types depending on target platform: generic on Windows (custom implementation that doesn't use high level OSs controls), gtk tree view on Linux-based platforms, and `NSOutlineView` on MacOS. They are all react differently on user input, and not exactly as we want it to. 
`wxDataView*Renderer` implementations are also platform-dependent, and class hierarchies they belong to are also different depending on platform.

> Are your custom implementations inferior

No doubt 🤣 

>  If not, which not just keeping them and remove the #if ?

I think I answered that question?